sap/camel-sap-starter/pom.xml

```diff
@@ -1,181 +1,245 @@
-				<version>3.0.1</version>
-				<executions>
-					<execution>
-						<id>attach-sources</id>
-						<phase>process-classes</phase>
-						<goals>
-							<goal>jar</goal>
-						</goals>
-					</execution>
-				</executions>
-			</plugin>
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-dependency-plugin</artifactId>
-				<executions>
-					<execution>
-						<id>copy-jco-idoc-libs</id>
-						<phase>validate</phase>
-						<goals>
-							<goal>copy</goal>
-						</goals>
-						<configuration>
-							<stripVersion>true</stripVersion>
-							<outputDirectory>${lib.directory}</outputDirectory>
-							<artifactItems>
-								<artifactItem>
-									<groupId>com.sap.conn.jco</groupId>
-									<artifactId>sapjco3</artifactId>
-									<overWrite>true</overWrite>
-									<destFileName>sapjco3.jar</destFileName>
-								</artifactItem>
-								<artifactItem>
-									<groupId>com.sap.conn.idoc</groupId>
-									<artifactId>sapidoc3</artifactId>
-									<overWrite>true</overWrite>
-									<destFileName>sapidoc3.jar</destFileName>
-								</artifactItem>
-							</artifactItems>
-						</configuration>
-					</execution>
-				</executions>
-			</plugin>
-
-		</plugins>
-
-		<pluginManagement />
-	</build>
-
-	<distributionManagement>
-		<repository>
-			<id>jboss-releases-repository</id>
-			<name>JBoss Releases Repository</name>
-			<url>https://repository.jboss.org/nexus/service/local/staging/deploy/maven2</url>
-		</repository>
-		<snapshotRepository>
-			<id>jboss-snapshots-repository</id>
-			<name>JBoss Snapshots Repository</name>
-			<url>https://repository.jboss.org/nexus/content/repositories/snapshots</url>
-		</snapshotRepository>
-	</distributionManagement>
-	<repositories>
-		<repository>
-			<id>fusesource-third-party-internal</id>
-			<url>https://repository.jboss.org/nexus/content/repositories/fs-releases/</url>
-			<layout>default</layout>
-			<snapshots>
-				<enabled>true</enabled>
-			</snapshots>
-			<releases>
-				<enabled>true</enabled>
-			</releases>
-		</repository>
-		<repository>
-			<id>brewroot</id>
-			<name>Brew Repository</name>
-			<url>http://download.eng.bos.redhat.com/brewroot/repos/jb-fuse-6.2-build/latest/maven</url>
-			<snapshots>
-				<enabled>false</enabled>
-			</snapshots>
-			<releases>
-				<enabled>true</enabled>
-				<updatePolicy>never</updatePolicy>
-			</releases>
-		</repository>
-		<repository>
-			<id>jboss-ea</id>
-			<name>Fuse Early Access Repository</name>
-			<url>https://repository.jboss.org/nexus/content/groups/ea</url>
-			<snapshots>
-				<enabled>true</enabled>
-			</snapshots>
-			<releases>
-				<enabled>true</enabled>
-			</releases>
-		</repository>
-		<repository>
-			<id>redhat-ga-repository</id>
-			<url>https://maven.repository.redhat.com/ga</url>
-			<releases>
-				<enabled>true</enabled>
-			</releases>
-			<snapshots>
-				<enabled>false</enabled>
-			</snapshots>
-		</repository>
-	</repositories>
+    <dependency>
```

Comment:
shouldn't com.sap.conn.* dependencies be optional?

Reply:
I don't think these dependencies are optional to connect to SAP...

Reply:
I see, I'm probably lost due to https://github.com/jboss-fuse/camel-spring-boot/pull/165/files/82c088dce4a07e70bf2b05c4a24613b25dc07946#diff-a673accc30d879187ec7ab600baa9d1660096286d68c2313c36ebd21ad9ba7e1R164

Reply:
I though that in order to use camel-sap and camel-sap-starter you have to provide com.sap.conn. artifacts by yourself

Reply:
Sure in that case maybe the `provided` scope would make more sense ?

Reply:
yes! I meant provided, not optional. sorry!